.gitmodules

```diff
@@ -0,0 +1,10 @@
+[submodule "modules/TestEZ"]
+	path = modules/TestEZ
+	url = https://github.com/benbrimeyer/TestEZ
+	branch = extend-expect
+[submodule "modules/lemur"]
```

Comment:
Curious  why you chose to integrate Lemur instead of sticking with just TestEZ. Do the two co-exist well? Does Lemur just pick up the same spec files that TestEZ works off of?

Reply:
lemur is basically a Roblox mock and doesnt have any opinions about test cases.

Reply:
Since we dont have any way to execute lua code in a Roblox env from the command line, we can generate a virtual roblox env using lua mocks instead. This is what lemur does for us.

Reply:
Cool sounds good then